src/components/common/FileUploadPdf.vue

```diff
@@ -43,10 +38,20 @@ export default class FileUploadPdf extends Vue {
   readonly uploadToUrl!: (url: string, file: File, key: string, userId: string) => Promise<AxiosResponse>
 
   /** Component key, used to force it to re-render. */
-  protected count = 0
+  count = 0
 
   /** Custom errors messages, use to put component into manual error mode. */
-  protected errorMessages = [] as Array<string>
+  errorMessages = [] as Array<string>
+
+  private pdfjsLib: any
+
+  created () {
+    // NB: we load the lib and worker this way to avoid a memory leak (esp in unit tests)
+    // NB: must use require instead of import or this doesn't work
+    // NB: must use legacy build for unit tests not running in Node 18+
+    this.pdfjsLib = require('pdfjs-dist/legacy/build/pdf.js')
+    this.pdfjsLib.GlobalWorkerOptions.workerSrc = require('pdfjs-dist/legacy/build/pdf.worker.entry')
+  }
```

Comment:
Nice! Does it still work? Please call this out to Riyaz in a "Testing Notes" comment.

Reply:
Yup! I've tested upload a PDF when filing a court order. It looks OK. Not sure if I missed anything though. I'll add a note for Riyaz.

Reply:
Resolving.